authors/apps/articles/exceptions.py

```diff
@@ -0,0 +1,5 @@
+from rest_framework.exceptions import APIException
+
+class ArticleDoesNotExist(APIException):
+    status_code = 404
+    default_detail = "Requested Article is inexistent in your articles"
```

Comment:
Please change `inexistent` to `does not exist`. It is simpler which is what we should always strive for when writing APIs.